packages/ui/src/stories/MultiValueInput.stories.tsx

```diff
@@ -0,0 +1,64 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import React, { useState } from "react";
+import { expect, fn, userEvent, within } from "@storybook/test";
+
+import { MultiValueInput } from "../multivalue-input";
+
+const meta: Meta<typeof MultiValueInput> = {
+	title: "MultiValueInput",
+	component: MultiValueInput,
+	tags: ["autodocs"],
+	argTypes: {},
+	args: { onChange: fn(), values: [] },
+};
+export default meta;
+
+type Story = StoryObj<typeof MultiValueInput>;
+
+export const Base: Story = {
+	args: {},
+};
+
+export const WithValues: Story = {
+	args: { values: ["cat", "dog"] },
+};
+
+export const ReplacedPlaceholder: Story = {
+	args: { values: ["cat", "dog"], placeholder: "Add a value meow" },
+};
+
+export const Interactive: Story = {
+	args: {},
+	render: () => {
+		const [values, setValues] = useState<string[]>([
+			"cat",
+			"dog",
+			"squirrel",
+			"alligator",
+			"wolf",
+			"otter",
+			"seal",
+			"mouse",
+			"turtle",
+			"lion",
+			"tiger",
+			"bobcat",
+		]);
+		return <MultiValueInput values={values} onChange={setValues} />;
+	},
+	play: async ({ canvasElement }) => {
```

Comment:
I was curious about storybook component testing so I added this. I think it's okay but seems a little buggy, or I'm doing something wrong, since this test doesn't pass on first render but always passes later. Anyway, this isn't automated at all, but may be worth looking more into if we do more storybook based dev